bin/rubocop-diff

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/sh
 
-function extract_offences() {
+extract_offenses() {
   echo "$1" | awk '{print $4}' | sed 's/no/0/g'
 }
 
-RED='\033[0;31m'
-GREEN='\033[0;32m'
-NC='\033[0m'
-
 current_branch=$(git branch | grep '\*' | cut -d ' ' -f2)
 
+unset BUNDLE_PATH
+unset BUNDLE_APP_CONFIG
+bundle install
+
 echo Running rubocop against current branch
```

Comment:
I'd recommend having the `echo` argument quoted here & below
```
echo 'Running rubocop against current branch'
```
or better, for reasons of portability, [use `printf` instead of `echo`](https://unix.stackexchange.com/questions/65803/)